frontend/src/pages/Visitors.vue

```diff
@@ -138,12 +138,9 @@
                 formatLineChartData:GET_FORMAT_LINE_CHART_DATA,
             }),
             units() {
-                switch (this.currentActiveButton) {
-                case BOUNCE_RATE:
+                if (this.currentActiveButton === BOUNCE_RATE) {
```

Comment:
Why did you delete this?

Reply:
because on page page view we don use units for avg time, and I delete here units for consistency